java/adapter/jdbc/src/main/java/org/apache/arrow/adapter/jdbc/consumer/DateConsumer.java

```diff
@@ -30,11 +34,28 @@
  */
 public class DateConsumer {
 
+  /**
+   * The number of milli-seconds in a day.
+   */
+  public static final long MILLIS_PER_DAY = TimeUnit.DAYS.toMillis(1);
```

Comment:
Isn't there a  TimeUnit.MILLIS.toDays that you can use instead of doing the division?

Reply:
Revised accordly. Thanks for the good suggestion.